framework/src/security/security_utils.c

```diff
@@ -267,11 +267,8 @@ int secutils_convert_path_s2h(const char *path, uint32_t *slot)
 	if (!strncmp(path, SS_PATH, sizeof(SS_PATH) - 1)) {
 		*slot = atoi(&path[3]);
 		return 0;
-	} else if (!strncmp(path, FACTORY_PATH, sizeof(FACTORY_PATH) - 1)) {
```

Comment:
Is this change within the scope of the commit description?

Reply:
it cause build fail